.github/workflows/deploy-binance.yml

```diff
@@ -0,0 +1,148 @@
+name: Deploy to Binance smart chain
+
+# Run on pushes to master or PRs to master
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Could you remove the branches condition here? So that we deploy this on every PR (for now, for QA).